features/netsocket/emac-drivers/TARGET_ARM_FM/COMPONENT_LAN91C111/fvp_emac.cpp

```diff
@@ -246,7 +232,7 @@ bool fvp_EMAC::power_up()
     mbed::mbed_event_queue()->call(mbed::callback(this, &fvp_EMAC::phy_task));
 
     /* Allow the PHY task to detect the initial link state and set up the proper flags */
-    osDelay(10);
+    wait_ms(10);
```

Comment:
Just because I know `wait` APIs are being a pain, I'd actually prefer going straight to `ThisThread::sleep_for(10)` here, given that you're relying on lots of other `rtos` APIs anyway.

Reply:
`wait_ms()` been updated to `ThisThread::sleep_for()`